middleware/json_loader.py

```diff
@@ -1,14 +1,18 @@
 import json
 import falcon
 
+
 class JsonLoader:
 
     def process_resource(self, req, res, resource, params):
+        # dont parse json if we just want the status
+        if type(resource).__name__ == "Metastatus":
```

Comment:
define Metastatus in a list of non-JSON routes.

```python
if resource.__class__.__name__ in non_json_routes:
```

Reply:
Although they are literally the same thing, `type` feels like reflection, and `.__class__` feels like OOP